examples/cli/src/main/java/io/unitycatalog/cli/utils/Oauth2CliExchange.java

```diff
@@ -207,23 +194,60 @@ private int findAvailablePort() throws IOException {
     }
   }
 
-  private String buildTokenBody(Map<String, String> tokenParams) throws JsonProcessingException {
-    return tokenParams.entrySet().stream()
-        .filter(
-            p -> !p.getKey().equals(Fields.CLIENT_ID) && !p.getKey().equals(Fields.CLIENT_SECRET))
-        .map(
-            p ->
-                URLEncoder.encode(p.getKey(), StandardCharsets.UTF_8)
-                    + "="
-                    + URLEncoder.encode(p.getValue(), StandardCharsets.UTF_8))
-        .reduce((p1, p2) -> p1 + "&" + p2)
-        .orElse("");
-  }
+  public static class URLEncodedForm {
```

Comment:
wondering if we can get rid of all this in favor of `toUrlQueryString()` on the generated code?